.devilbox/www/htdocs/vhosts.php

```diff
@@ -84,7 +84,7 @@ function updateStatus(vhost) {
 						} else {
 							el_valid.className += ' bg-success';
 							el_valid.innerHTML = 'OK';
-							el_href.innerHTML = '<a target="_blank" href="http://'+vhost+'.<?php echo $Docker->getTld();?>">'+vhost+'.<?php echo $Docker->getTld();?></a>';
+							el_href.innerHTML = '<a target="_blank" href="http://'+vhost+'.<?php echo $Docker->getTld().$Docker->getPort();?>">'+vhost+'.<?php echo $Docker->getTld();?></a>';
```

Comment:
I'd say, you also add the port to the display itself. `<a href="...:8080">also here I would say</a>`:
```
href="http://'+vhost+'.<?php echo $Docker->getTld().$Docker->getPort();?>">'+vhost+'.<?php echo $Docker->getTld().$Docker->getPort();?></a>';
```
What do you think?